src/main/webapp/js/adminSearch.js

```diff
@@ -1,6 +1,10 @@
 $(document).ready(function() {
     $(".fslink").hide();
 
+    var searchKey = $('#filterQuery').val();
+    var splittedSearchKey = searchKey.split(" ");		/* Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )*/
+    $("body").highlight(splittedSearchKey);
```

Comment:
Could you just highlight words inside the body of the result tables only? We only have 2 tables `#search_table_instructor` and `#search_table`.